tests/testthat/test-rmd_to_md.R

```diff
@@ -1,100 +1,69 @@
-test_that("rmd_to_md() writes .md to a directory", {
-  skip_if_offline()
-  rmd_file <- "https://raw.githubusercontent.com/b-cubed-eu/gcube/refs/heads/main/vignettes/articles/occurrence-process.Rmd"
+test_that("example.Rmd file is accessible", {
+  file_path <- testthat::test_path("example.Rmd")
+  expect_true(file.exists(file_path))
+})
 
+test_that("rmd_to_md() writes .md and figures to the expected directories", {
   temp_dir <- tempdir()
-
-  md_dir <- file.path(temp_dir, "src", "content", "docs", "r", "gcube")
-  fig_dir <- file.path(temp_dir, "public", "r", "gcube")
-  fig_url_dir <- paste0(temp_dir, "/astro-docs/", "r", "gcube", "/")
-  order <- 1
-
-  rmd_to_md(rmd_file, md_dir, fig_dir, fig_url_dir, order)
+  on.exit(unlink(temp_dir, recursive = TRUE))
+  expected_md_dir <- file.path(temp_dir, "src/content/docs/software/example")
+  expected_fig_dir <- file.path(temp_dir, "public/software/example")
+
+  rmd_to_md(
+    rmd_file = testthat::test_path("example.Rmd"),
+    md_dir = expected_md_dir,
+    fig_dir = expected_fig_dir,
+    fig_url_dir = "/software/example/",
+    sidebar_order = 2
```

Comment:
Please remove this line (and comma above) once `sidebar_order` is no longer required.